serverlessworkflow/modules/ROOT/pages/persistence/postgresql-flyway-migration.adoc

```diff
@@ -9,30 +9,31 @@ Kogito provides an out-of-the-box way to manage your database schema changes wit
 
 When you upgrade your Kogito version, by default it will pick up the latest changes available from Kogito and apply them automatically.
 
-NOTE: This feature is currently available only for `kogito-addons-quarkus-persistence-postgresql` extension.
-
 == How to migrate
 
 === Migrate using Flyway Config
 
 * Kogito provides a default mechanism for migrating your database while updating the Kogito version using the following `Flyway` properties (default value is `false`):
 +
 --
-[source,console]
+[source,properties]
 ----
 quarkus.flyway.migrate-at-start=true
+quarkus.datasource.db-kind=postgresql
 ----
 --
 This will create a schema history table `flyway_schema_history` in your database to track the version of each database, recording in it every versioned migration file applied to build that version.
 
+NOTE: When using `kogito-addons-persistence-jdbc`, it is mandatory to set the `quarkus.datasource.db-kind` property, so thay Flyway can locate the appropriate scripts for the database.
```

Comment:
```suggestion
NOTE: When using `kogito-addons-persistence-jdbc`, it is mandatory to set the `quarkus.datasource.db-kind` property, so that Flyway can locate the appropriate scripts for the database.
```